tests/deploy_account.rs

```diff
@@ -0,0 +1,140 @@
+#![deny(warnings)]
+
+use felt::felt_str;
+use starknet_rs::{
+    business_logic::{
+        execution::objects::{CallInfo, CallType, TransactionExecutionInfo},
+        fact_state::in_memory_state_reader::InMemoryStateReader,
+        state::{cached_state::CachedState, state_api::State},
+        transaction::objects::internal_deploy_account::InternalDeployAccount,
+    },
+    definitions::{general_config::StarknetChainId, transaction_type::TransactionType},
+    services::api::contract_class::{ContractClass, EntryPointType},
+    utils::{felt_to_hash, Address},
+};
+use std::path::PathBuf;
+
+#[test]
+fn internal_deploy_account() {
+    let state_reader = InMemoryStateReader::new(Default::default(), Default::default());
+    let mut state = CachedState::new(state_reader, None);
+
+    state.set_contract_classes(Default::default()).unwrap();
+
+    let class_hash = felt_to_hash(&felt_str!(
+        "3146761231686369291210245479075933162526514193311043598334639064078158562617"
+    ));
+    let contract_class = ContractClass::try_from(PathBuf::from(
+        "starknet_programs/account_without_validation.json",
+    ))
+    .unwrap();
+
+    state
+        .set_contract_class(&class_hash, &contract_class)
+        .unwrap();
+
+    let internal_deploy_account = InternalDeployAccount::new(
+        class_hash,
+        0,
+        0, // TODO: Value 340282366920938463463374607431768211457 is too large.
```

Comment:
Instead of adding this TODO i think it is better to create two tickets in Trello and remove them from here.

Reply:
I left it there to have a reference of what it should be once we fix it.
Maybe we could create the tickets but leave it there for whoever has to fix it.